vite.config.ts

```diff
@@ -52,6 +53,92 @@ export default defineConfig(({ mode }) => {
       }),
       react(),
       svgr(),
+      VitePWA({
+        registerType: "autoUpdate",
+        devOptions: {
+          enabled: true,
+        },
+        workbox: {
+          maximumFileSizeToCacheInBytes: 3097152,
+          globPatterns: ["**/*.{js,css,html,ico,png,svg,gif,hex}"],
+          runtimeCaching: [
+            {
+              urlPattern: /^https:\/\/ajwvhvgo.apicdn.sanity.io\/.*/,
```

Comment:
Only worth addressing after other issues: we've parameterised the project name and dataset elsewhere so should do that here too (and below).